carma_wm_ctrl/src/WMBroadcasterNode.cpp

```diff
@@ -79,7 +79,9 @@ int WMBroadcasterNode::run()
 
   pnh2_.getParam("/config_speed_limit", config_limit);
   wmb_.setConfigSpeedLimit(config_limit);
-  
+  timer = pnh_.createTimer(ros::Duration(10.0),
+   [this](const auto&){control_msg_pub_.publish(&WMBroadcaster::controlRequestFromRoute);});
```

Comment:
I'm amazed this is building since a pointer to a method is being passed to a publisher.
You need to call the method controlRequestFromRoute with the most recent route message. Then you would publish the traffic control request based on that message.